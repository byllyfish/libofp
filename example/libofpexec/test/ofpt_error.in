{params: {type: OFPT_ERROR, version: 1, msg: {type: 1, code: 1, data: '0123456789'}}, method: ofp.send, id: 1}
{params: {type: OFPT_ERROR, msg: {type: 1, code: 1, data: '0123456789'}}, method: ofp.send, id: 2}
{params: {type: OFPT_ERROR, version: 2, msg: {type: 1, code: 1, data: '0123456789'}}, method: ofp.send, id: 3}
{params: {type: OFPT_ERROR, version: 3, msg: {type: 1, code: 1, data: '0123456789'}}, method: ofp.send, id: 4}
{params: {type: OFPT_ERROR, version: 4, msg: {type: 1, code: 1, data: '0123456789'}}, method: ofp.send, id: 5}
