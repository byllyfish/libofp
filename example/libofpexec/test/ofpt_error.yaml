---
type:            OFPT_ERROR
xid:             0x00000000
version:         0x01
msg:             
  type:            1
  code:            1
  data:            0123456789
...
---
type:            OFPT_ERROR
xid:             0x00000000
version:         0x04
msg:             
  type:            1
  code:            1
  data:            0123456789
...
---
type:            OFPT_ERROR
xid:             0x00000000
version:         0x02
msg:             
  type:            1
  code:            1
  data:            0123456789
...
---
type:            OFPT_ERROR
xid:             0x00000000
version:         0x03
msg:             
  type:            1
  code:            1
  data:            0123456789
...
---
type:            OFPT_ERROR
xid:             0x00000000
version:         0x04
msg:             
  type:            1
  code:            1
  data:            0123456789
...
