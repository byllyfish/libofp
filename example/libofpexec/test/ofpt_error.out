---
event:           LIBOFP_LOOPBACK
params:          
  data:            0101001100000000000100010123456789
...
---
event:           LIBOFP_LOOPBACK
params:          
  data:            0101001100000000000100010123456789
...
---
event:           LIBOFP_LOOPBACK
params:          
  data:            0201001100000000000100010123456789
...
---
event:           LIBOFP_LOOPBACK
params:          
  data:            0301001100000000000100010123456789
...
---
event:           LIBOFP_LOOPBACK
params:          
  data:            0401001100000000000100010123456789
...
